launch_bluesky_headless.sh

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/bash
+set -e
+set -o xtrace
+
+if [ "$1" != "" ]; then
+    imagename=$1
+else
+    imagename="bluesky"
+fi
+
+podman run --pod acquisition \
+       -ti  --rm \
+       -v `pwd`:'/app' -w '/app' \
+       -v ./bluesky_config/ipython:/usr/local/share/ipython \
+       -v ./bluesky_config/databroker:/usr/local/share/intake \
+       -v ./bluesky_config/happi:/usr/local/share/happi \
+       -e XDG_RUNTIME_DIR=/tmp/runtime-$USER \
```

Comment:
Is this needed in headless mode? (Is a QApplication started?)

Reply:
Works without it, thanks!